homeassistant/components/switchbot_cloud/__init__.py

```diff
@@ -111,6 +113,10 @@ def make_device_data(
             devices_data.locks.append(
                 prepare_device(hass, api, device, coordinators_by_id)
             )
+        if isinstance(device, Device) and device.device_type in ["Bot"]:
+            devices_data.buttons.append(
```

Comment:
`buttons` is never used. the `button` platform is using `switches` instead. is this needed?

Reply:
It was a mistake on my part, I've updated the `button` platform code to use `buttons`